recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -12,8 +18,9 @@ class MsdfAtlasGenConan(ConanFile):
     description = "MSDF font atlas generator"
     topics = ("msdf-atlas-gen", "msdf", "font", "atlas")
     settings = "os", "arch", "compiler", "build_type"
+    package_type = "application"
 
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeToolchain"
     exports_sources = ["CMakeLists.txt"]
```

Comment:
If the project already uses CMake, then you should most likely delete the CMakeLists.txt in the recipe folder and drop this attribute. It's no longer needed with the new generators in Conan v2.